WordPress/Classes/Services/MediaExportService.swift

```diff
@@ -182,6 +199,7 @@ open class MediaExportService: LocalCoreDataService {
     /// Configure Media with the AssetExport.
     ///
     fileprivate func configureMedia(_ media: Media, withExport export: MediaAssetExporter.AssetExport) {
+        media.remoteStatusNumber = NSNumber(value: MediaRemoteStatus.local.rawValue)
```

Comment:
I don't feel like this belongs in this method. Everything else in this class is just to do with exporting the media, and the remote status isn't really related.

I feel like it'd make more sense in `makeMedia(in:)` in `Media+Blog`, as a kind of default value for the status (we also set the mediaID to zero and the creation date in there).